apps/backoffice/openapi.yaml

```diff
@@ -879,6 +879,37 @@ paths:
           description: Too many requests
         '500':
           description: Internal server error
+  /assistance:
+    post:
+      tags:
+        - assistance
+      summary: Zendesk integration
+      description: Integrate the possibility of requesting assistance by indicating an email on which to receive feedback
+      operationId: assistance
+      security:
+        - cookieAuth: []
+      requestBody:
+        description: Email on which to receive feedback
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/AssistanceItem'
+        required: true
+      responses:
```

Comment:
You must insert 400 bad request response (in case of request payload validation error)

Reply:
[10750c1](https://github.com/pagopa/io-services-cms/pull/666/commits/10750c139e3f40cfc9e95b34fdbc62059cdcc7b5)